src/components/TradesTable.tsx

```diff
@@ -1,10 +1,11 @@
+import type { BonfidaTrade } from '../utils/types';
```

Comment:
I believe that this prevents the actual _code_ from being pulled in, when all this module needs is the Typescript type. That's how it works in Flow anyway.